package.json

```
@@ -27,11 +27,12 @@
     "test:unit:coverage": "yarn test:unit && open coverage/lcov-report/index.html",
     "test:karma": "yarn build-npm && karma start",
     "prebuild": "node -p \"'export const LIB_VERSION = ' + JSON.stringify(require('./package.json').version) + ';'\" > src/version.ts",
-    "build": "node compile-assets.js && webpack --config webpack.config.js",
+    "compile": "node compile-assets.js",
```

```suggestion
    "prebuild": "node compile-assets.js",
```
unless there's a reason to run build without compile?